src/raftstore/store/fsm/apply.rs

```diff
@@ -1731,6 +1731,14 @@ impl ApplyDelegate {
             ));
         }
 
+        info!(
+            "{} execute admin command {:?} at [term: {}, index: {}]",
+            self.tag,
+            req,
+            ctx.exec_ctx.as_ref().unwrap().term,
+            ctx.exec_ctx.as_ref().unwrap().index;
```

Comment:
Oh, can't believe that `;` will work here.

Reply:
Yeah, macros from slog supports this.

Reply:
But the `;` is not necessary here, please remove it.